### ISSUE TITLE
this is a rat

### ISSUE THREAD

**Issue by croosanat** [vanishgg/VanishRaider#6]
dont dowload this

**Comment by vanishgg** [comment]
yes so true its an obv trojan, logger, dropper. and i would not downloading this or running it very unsafe i scanned on "vIrUsToTaL" and it said it was malware.. 

dont dowload this..

**Comment by vanishgg** [comment]
> It is a trojan + rat. I scanned it in Cuckoo scan virus, and it got a score of 10/10, which Very dangerous. They will secretly remote control your PC. You will get a backdoor installed in Windows 10 -11. Do not download this.

all true i have access to 250+ pcs
